Week 4/Extra Practice Problems (Week 4)/2405: Optimal Partition of String/dominic.cpp

```diff
@@ -0,0 +1,36 @@
+// Fun and faster (but a bit more complicated) bitwise
+class Solution {
+public:
+    int partitionString(string s) {
+        int n = 0;
+        int res = 1;
+        for (char &c : s) {
+            int curr = (2 << (c-'a'));
```

Comment:
why not shift 1?

Reply:
Oh, I meant to put 1 there. Nice catch
```suggestion
            int curr = (1 << (c-'a'));
```